diff -up notify-0.3.1/setup.py.omv~ notify-0.3.1/setup.py
--- notify-0.3.1/setup.py.omv~	2022-08-08 15:37:36.798127979 +0200
+++ notify-0.3.1/setup.py	2022-08-08 15:37:48.089841936 +0200
@@ -17,10 +17,6 @@ def readlist(filename):
     rows = [x.strip() for x in rows if x.strip()]
     return list(rows)
 
-extras = {}
-if sys.version_info >= (3,):
-    extras['use_2to3'] = True
-
 setup(
     name = NAME,
     version = VERSION,
@@ -59,6 +55,5 @@ setup(
         'console_scripts': [
             'notify = notify.console:main',
         ],
-    },
-    **extras
+    }
 )
